lib/ce/src/hot-settings-resolver.service.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable, SimpleChanges } from '@angular/core';
 
-const AVAILABLE_OPTIONS: string[] = ['allowEmpty', 'allowHtml', 'allowInsertColumn', 'allowInsertRow',
+const AVAILABLE_OPTIONS: string[] = ['activeHeaderClassName', 'allowEmpty', 'allowHtml', 'allowInsertColumn', 'allowInsertRow',
```

Comment:
This shouldn't be replaced with 
```
const AVAILABLE_OPTIONS: string[] = Object.keys(Handsontable.DefaultSettings.prototype);
const AVAILABLE_HOOKS: string[] = Object.keys(Handsontable.hooks.globalBucket);
```
as in pro version?

Reply:
We have to fix our definition file first.